components/crud-web-apps/jupyter/backend/apps/common/form.py

```diff
@@ -135,6 +135,25 @@ def set_notebook_image_pull_policy(notebook, body, defaults):
     )
 
 
+def set_notebook_http_rewrite_uri(notebook, body, defaults):
```

Comment:
We can remove this function, the backend can set the necessary annotations by looking at the server type